README.md

```diff
@@ -7,3 +7,40 @@
 # WiX Toolset v4
 
 This repository contains the WiX Toolset v4 codebase.
+
+# Developing WiX
+
+## Prerequisites
+
+Visual Studio 2022 (17.0.4 or higher, NOT the preview release) with the following installed:
+
+| Workloads |
+| :-------- |
+|ASP.Net and web development |
+|.Net desktop development |
+| Desktop development with C++ |
+
+| Individual components | Notes |
+| :-------- |
+|.Net Framework 4.7.2 SDK | |
+|.Net Framework 4.7.2 targeting pack | |
+| MSVC v141 - VS 2017 C++ ARM64 build tools (v14.16) | |
+| MSVC v142 - VS 2017 C++ x64/x86 build tools (v14.16) | |
+| MSVC v142 - VS 2019 C++ ARM64 build tools (v14.29-16.11) | Required for WixE2E.WixE2EFixture.CanBuildPackageWithNativeWindowsApp |
+| MSVC v142 - VS 2019 C++ x64/x86 build tools (v14.29-16.11) | Required for WixE2E.WixE2EFixture.CanBuildPackageWithNativeWindowsApp |
```

Comment:
These three lines should no longer be necessary with #98

Reply:
Correct, and they are already fixed locally.